docs/docs/workflows/troubleshooting.md

```diff
@@ -55,14 +55,12 @@ When in doubt, check if the encountered [issue is known](https://github.com/edge
 
 :::
 
-
 :::tip
 
-During an upgrade with modified attestation config, a backup of the current configuration is stored in the `join-config-backup` config map in the `kube-system` namespace. To restore the old attestation config after a failed upgrade, you can copy the attestation config from this resource, put it in your configuration file and retry the upgrade.
+During an upgrade with modified attestation config, a backup of the current configuration is stored in the `join-config` config map in the `kube-system` namespace under the `attestationConfig_backup` key. To restore the old attestation config after a failed upgrade, replace the value of `attestationConfig` with the value from `attestationConfig_backup`. Alternatively you can copy the attestation config from `attestationConfig_backup`, put it in your configuration file and rerun `constellation upgrade apply`.
```

Comment:
How about adding the kubectl commands to replace attestationConfig with the backup?

Reply:
I think letting the user change the config is safer because subsequent `upgrade apply` will try to apply the wrong config again.

Reply:
This would require the user to convert the attestation config from JSON to YAML however.
If a user simply runs `constellation config fetch-measurements` + `constellation upgrade apply`, this problem should never occur in the first place